MainContainer/client/assets/maincontainer-style.css

```diff
@@ -367,6 +339,20 @@ a.active-yellow span {
   width: 20%;
   height: auto;
 }
+.intro-box {
+  width: 82%;
+  margin: 0 auto;
+  box-shadow: 0px 3px 6px #00000029;
```

Comment:
Nitpick: Improper hex value, should be #000029

Reply:
The hex value should be alright, the two more numbers at the end are used to define the alpha value

Reply:
Whoops, I meant it should be `#000029 `, Github just switches it to an issue reference. Hex values should be 6 digits in total.

Reply:
According to [Mozilla docs](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value), hex with alpha value is using 8 digits while normal hex is using 6 digits. On the design, the box-shadow is a little bit translucent so I used the alpha value. Did Github issue reference only complain here? I also used the 8 digits hex on "image-tint"

Reply:
Interesting, I had no idea that you could do that. Just read about it [here](https://alligator.io/css/hex-code-colors-alpha-values/#adding-an-alpha-value-to-your-hex-codes). 

I've only seen transparency adjusted this way with rgba and a 4th value:
```css
box-shadow: 10px 10px 10px rgba(0, 0, 0, 0.5);
```
Then using Hex with alpha value is alright - if it's repeated everywhere, we should probably eventually write it down in a style guide for the VPIA.